### ISSUE TITLE
Version of VPAT template in use in ICDS and referenced in Accessibility Statement / Accessibility Compliance Report documentation is inconsistent

### ISSUE THREAD

**Issue by MI6-94** [mi6/ic-design-system#334]
## Summary of the bug

Version of VPAT template in use in ICDS and referenced in Accessibility Statement / Accessibility Compliance Report documentation is inconsistent.
 
**Version inconsistency**
 
There is a version inconsistency/format inconsistency between "internal" IC authored VPAT versions and public available VPAT referenced as being from ITIC.

The IC authored templates at [blank accessibility statement](https://design.sis.gov.uk/accessibility-statement-template.docx) and [blank accessibility compliance report](https://design.sis.gov.uk/blank-accessibility-conformance-report-vpat-wcag-24rev.docx) are shortened versions which reference the ITIC versions.

This is inconsistent with, for example, [EN 301 549 blank ACR template](https://design.sis.gov.uk/blank-accessibility-conformance-report-vpat-en301549-24rev.docx) which is drawn directly from the upstream ITIC page at https://www.itic.org/policy/accessibility/vpat
 
**ICDS as public document / public URLs**
 
Given that the ICDS is now a public document developed on GitHub with embedded URLs, the internally written VPAT should be deprecated in favour of publicly available documents. This may have additional effects on documentation if URLs are embedded in the Digital Accessibility Policy or elsewhere.

This also applies to the ICDS's own accessibility statement (which includes the optional covering page which states that it can be deleted).
